### ISSUE TITLE
Error in generating large sketches

### ISSUE THREAD

**Issue by laserson** [marbl/Mash#53]
I'm trying to generate a sketch with `-s 100000` on about 7000 RefSeq genomes (using 21-mers).  The program finishes fine (with an output of about 2.1 GB) but when I run `mash info refseq.msh` I get the following error.  I was able to successfully build a sketch with 50000 hashes.

```
libc++abi.dylib: terminating with uncaught exception of type kj::ExceptionImpl: kj/io.c++:305: failed: miniposix::read(fd, pos, max - pos): Invalid argument; fd = 3
stack: 0x10a648e73 0x10a641425 0x10a624229 0x10a61d777 0x10a61ccae 0x10a612eca 0x10a60ab4c 0x10a617b0a 0x10a61bcc8 0x10a604794 0x3
Abort trap: 6
```

**Comment by ondovb** [comment]
I believe this is the same issue as #16. It should be fixed in master if you are able to build from source; otherwise a fix will be in the next release.

**Comment by ondovb** [comment]
Fixed in [v2.0](https://github.com/marbl/Mash/releases/tag/v2.0). Please reopen if issues persist.
